tests/csrf_tests/tests.py

```diff
@@ -189,6 +189,17 @@ def test_csrf_token_in_header(self):
         req2 = CsrfViewMiddleware().process_view(req, post_form_view, (), {})
         self.assertIsNone(req2)
 
+    def test_csrf_token_in_header_with_customized_name(self):
+        """
+        Check that we can pass in the token in a header with customized name
```

Comment:
I find this docstring a bit confusing (copied from elsewhere, I know). I think something like "settings.CSRF_HEADER_NAME can be used to customize the CSRF header name" would be simpler